lms/static/js/learner_dashboard/views/program_card_view.js

```diff
@@ -23,6 +25,31 @@
                 render: function() {
                     var templated = this.tpl(this.model.toJSON());
                     this.$el.html(templated);
+                    this.postRender();
+                },
+                postRender: function() {
+                    if(navigator.userAgent.indexOf('MSIE') !== -1 ||
+                        navigator.appVersion.indexOf('Trident/') > 0){
+                        /* Microsoft Internet Explorer detected in. */
+                        window.setTimeout( function() {
+                            this.reLoadBannerImage();
+                        }.bind(this), 100);
+                    }
+                },
+                // Defer loading the rest of the page to limit FOUC
+                reLoadBannerImage: function() {
+                    var $img = this.$('.program_card .banner-image'),
+                        imgSrcAttr = $img ? $img.attr('src') : {};
+                    
+                    if (!imgSrcAttr || imgSrcAttr.length < 0) {
+                        try{
+                            picturefill({
+                                reevaluate: true
+                            });
+                        }catch(err){
+                            //Swallow the error here
+                        }
+                    }
```

Comment:
It's a good point. I don't know if the frontend unit test framework `jasmine` allows the set of different `navigator.userAgent` so I can test this. By the way, these functionality is to support IE10, which maybe a browser going away. I don't think devote too much time to the correctness of this code is that necessary